cves/CVE-2016-6795.yml

```
@@ -45,7 +45,7 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: The URL provided by the user was not fully validated which could cause the server to let an attacker go to locations not permitted to view and even inject code to execute remotely
```

```suggestion
description: |
   The URL provided by the user was not fully validated which could cause the server to let an attacker go to locations 
   not permitted to view and even inject code to execute remotely
```
How could the crafted URL remotely execute code? A little more detail here may be nice.

---

Mainly just agreeing with Matt about the change to the formatting. Adding the pipe character will make this much easier to read.